### PR TITLE
More info about how to run docker compose in dev environment

### PR DESCRIPTION
**Pull request by viktorprogger**

### CODE REVIEW

**Comment by s1lver** [comment]
Maybe move it to a separate document?
e.g 

> - /docs
>   + /img 
>     + img.png
>   + docker.md

**Comment by vjik** [comment]
> Maybe move it to a separate document? e.g
> 
> > * /docs
> >   
> >   * /img
> >     
> >     * img.png
> >   * docker.md

Good idea. For example, in validator: https://github.com/yiisoft/validator/blob/master/docs/internals.md

File: README.md [hunk 1]
```diff
@@ -38,6 +38,32 @@ to the `require` section of your `composer.json` file.
 ## Extra
 
 ### Docker
+
+_Don't forget you have to have the latest version of `docker` installed and running on your computer in order to use it._
+
+#### Developer environment
+This project has ready to use docker environment inside the `tests` directory.
```
Comment (samdark) [inline README.md:45]:
```suggestion
#### Developer environment

This project has ready to use docker environment inside the `tests` directory.  
```

File: README.md [hunk 2]
```diff
@@ -38,6 +38,32 @@ to the `require` section of your `composer.json` file.
...
+`docker-compose.yml` with the additional config file `docker-compose.development.yml`. The simplest way is
+to copy `.env.example` file with name `.env`. It has essential environment variables to use both these `yml` configurations.
+Review these variables and change if needed.
+
+Then you can use docker compose as always. To execute unit tests, run the command
+`docker compose run --rm php82 vendor/bin/phpunit` inside the `tests` folder.
+You can also choose the needed version of php instead of 8.2.
+
+#### xDebug and PhpStorm
+For xDebug to work properly with PhpStorm, you have to additionally configure your PhpStorm.
```
Comment (samdark) [inline README.md:58]:
```suggestion
#### xDebug and PhpStorm

For xDebug to work properly with PhpStorm, you have to additionally configure your PhpStorm.
```

**Review by samdark — APPROVED** [review]

**Review by s1lver — APPROVED** [review]

**Pull request merged**